### DISCUSSION TITLE
Newbie here

### DISCUSSION THREAD

**Discussion by RoyGila** [community/community#49786] [Copilot Conversations]
Where do i start?

**Comment by enviGit** [comment]
### Welcome to GitHub [RoyGila](https://github.com/RoyGila)! 

Here are some basic instructions to get you started:


- **Create a new repository:** 
Once you've signed up, you can create a new repository to store your code. Click on the "New" button on your dashboard, and give your repository a name.
- **Set up Git on your computer:** 
To use GitHub, you'll need to have Git installed on your computer. Go to [Git website](https://git-scm.com/) and download the appropriate version for your operating system. Install Git and follow the instructions.
- **Clone the repository:** 
Once you have Git installed, you'll need to clone your repository onto your computer. To do this, copy the URL of your repository from the GitHub website and open up a terminal window on your computer. Navigate to the folder where you want to store your code, and type the following command:
`git clone [repository URL]` 
_This will create a local copy of your repository on your computer._
- **Make changes:** 
Now that you have a local copy of your repository, you can make changes to your code. Open up your favorite code editor and make changes to your files.
- **Stage your changes:** 
Once you've made changes to your code, you'll need to stage them for commit. In your terminal window, navigate to your local repository and type the following command:
`git add .` 
_This will stage all of the changes you've made to your code._
- **Commit your changes:** 
After you've staged your changes, you'll need to commit them to your repository. Type the following command:
`git commit -m "your commit description"`
_This will create a commit with your changes and a message describing what you've done._
- **Push your changes:** 
Finally, you'll need to push your changes to the GitHub website. Type the following command:
`git push`
_This will upload your changes to your repository on GitHub._

And that's about it, you've just made your first contribution to GitHub! From here, you can continue to make changes to your code, commit them, and push them to your repository on GitHub. You can also explore other repositories on the site, collaborate with other developers, and contribute to open source projects.

_If you need help with a specific topic feel free to ask._

**Comment by Ntshangase** [comment]
Hello, I would say build your GitHub profile, Check out this repo for more: https://github.com/Ntshangase/GitHub-Profile-Builder

**Comment by iamcjwilliams** [comment]
@RoyGila Here is another helpful doc: https://docs.github.com/en/copilot/getting-started-with-github-copilot
